### PR TITLE
Browser context tests

### LINKED ISSUE #5: Add browsercontext.spec.js tests
Based on this tag https://github.com/kblok/playwright/tree/first-snapshot. We don't need to get the tests green. We only need to incoporate the code. Let's also check if we can get code from Puppeteer

### PR DESCRIPTION
**Pull request by kblok**
closes #5

### CODE REVIEW

File: src/PlaywrightSharp.Abstractions/IFrame.cs [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        /// <summary>
+        /// Executes a function in browser context
+        /// </summary>
+        /// <param name="script">Script to be evaluated in browser context</param>
+        /// <param name="args">Arguments to pass to script</param>
+        /// <remarks>
+        /// TODO
+        /// </remarks>
+        /// <returns>Task that completes result of the script</returns>
+        Task<T> EvaluateAsync<T>(string script, params object[] args);
```
Comment (Meir017) [inline src/PlaywrightSharp.Abstractions/IFrame.cs]: maybe we should use the same approach as in puppeteer-sharp and have an `EvaluateFunction` & `EvaluateExpression` methods

File: src/PlaywrightSharp.Abstractions/PlaywrightSharp.Abstractions.csproj [hunk 2]
```diff
@@ -12,4 +12,7 @@
     <Commandlineparameters></Commandlineparameters>
     <RunWithWarnings>true</RunWithWarnings>
   </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
```
Comment (Meir017) [inline src/PlaywrightSharp.Abstractions/PlaywrightSharp.Abstractions.csproj]: maybe we should use `System.Json`?

File: src/PlaywrightSharp.Abstractions/IFrame.cs (hunk 1 above)
Comment (kblok) [reply src/PlaywrightSharp.Abstractions/IFrame.cs]: I had this conversation with the Playwright team https://github.com/microsoft/playwright/issues/710
We are going to try to go to one single method this time.

File: src/PlaywrightSharp.Abstractions/PlaywrightSharp.Abstractions.csproj (hunk 2 above)
Comment (kblok) [reply src/PlaywrightSharp.Abstractions/PlaywrightSharp.Abstractions.csproj]: That's a good one! 
I thought that was .NET Core. It's net standard :)

**Pull request merged**